src/scripts/main.js

```diff
@@ -2,7 +2,42 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach(person => {
+  const row = document.createElement('tr');
+  const fields = ['name', 'sex', 'born', 'died'];
+
+  fields.forEach(field => {
+    if (field === 'sex') {
+      row.appendChild(createCell(generateGenderTitle(person[field])));
+    } else {
+      row.appendChild(createCell(person[field]));
+    }
```

Comment:
It's better to use ternary operator here, as it will make a code shorter

Reply:
sure!